packages/legacy-client/src/ontology-runtime/ontologyProvider/OntologyProvider.ts

```diff
@@ -354,7 +359,7 @@ export class OntologyProvider {
     propertyName: string,
     body: StreamPointsBody,
   ): AsyncGenerator<Result<TimeSeriesPoint<T>, TimeSeriesError>, any, unknown> {
-    yield* wrapIterator(
+    yield* this.wrapIterator(
```

Comment:
I think you got a merge conflict, I already extracted wrapIterator, createPageIterator, and iterateLinkedObjects